rustls-mbedtls-provider-utils/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rustls-mbedtls-provider-utils"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
I think the minor version should be bumped, since we are switching to a different `rustls`

Reply:
sure, will do that